### DISCUSSION TITLE
Recieving multiple images instead of one from message with one image

### DISCUSSION THREAD

**Discussion by turdakhov** [def-studio/telegraph#411] [Q&A]
Helllo! This is my messages handler. 

```
class MyWebhookHandler extends WebhookHandler
{    
protected function handleChatMessage(Stringable $text): void
    {
        $photos = $this->message->photos();
        foreach ($photos as $photo) {
            $width = $photo->width();
            $this->chat->html("Width: $width")->send();
        }
    }
}
```
But when I send only one photo to the bot, this function returns 3-4 widths of photos. The $photos collection contains 3-4 Photo entities with own ids, heights, widths, and sizes.
Screenshot of the chat: https://vkosmos.kz/Screenshot_6.png
**Why it happens?**

When I open this Photos by id's (3 different ids), telegram returnes the same photos with the same sizes.

**Comment by fabio-ivona** [comment]
Hi, this is due to Telegram APIs, it returns an array of sizes:

https://core.telegram.org/bots/api#message
